edx_sga/static/js/src/edx_sga.js

```diff
@@ -204,6 +208,24 @@ function StaffGradedAssignmentXBlock(runtime, element) {
             });
         }
 
+        function updateChangeEvent(fileUploadObj) {
+            fileUploadObj.off('change').on('change', function (e, data) {
```

Comment:
*data* as function param is unused variable, Is it necessary to add this variable in param